src/components/ha-climate-state.html

```diff
@@ -37,17 +45,21 @@
   static get properties() {
     return {
       stateObj: Object,
+      currentStatus: {
+        type: String,
+        computed: 'computeCurrentStatus(stateObj)',
+      },
     };
   }
 
   computeCurrentStatus(stateObj) {
     if ('current_temperature' in stateObj.attributes) {
       return `${stateObj.attributes.current_temperature} ${stateObj.attributes.unit_of_measurement}`;
-    } else if ('current_humidity' in stateObj.attributes) {
+    }
+    if ('current_humidity' in stateObj.attributes) {
```

Comment:
I think that we should compare `stateObj.attributes.current_humidity != null`. That way we will guard against both `undefined` and `null` but not any other value.